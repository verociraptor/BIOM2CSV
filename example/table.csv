,Sample1,Sample2,Sample3,Sample4,Sample5,Sample6
GG_OTU_1,0.0,0.0,1.0,0.0,0.0,0.0
GG_OTU_2,5.0,1.0,0.0,2.0,3.0,1.0
GG_OTU_3,0.0,0.0,1.0,4.0,0.0,2.0
GG_OTU_4,2.0,1.0,1.0,0.0,0.0,1.0
GG_OTU_5,0.0,1.0,1.0,0.0,0.0,0.0
