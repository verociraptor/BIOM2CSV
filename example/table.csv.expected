Sample1,Sample2,Sample3,Sample4,Sample5,Sample6
0.0,0.0,1.0,0.0,0.0,0.0
5.0,1.0,0.0,2.0,3.0,1.0
0.0,0.0,1.0,4.0,0.0,2.0
2.0,1.0,1.0,0.0,0.0,1.0
0.0,1.0,1.0,0.0,0.0,0.0
